holidays/countries/belarus.py

```diff
@@ -44,35 +51,95 @@ def __init__(self, *args, **kwargs):
 
     def _populate_public_holidays(self):
         # New Year's Day.
-        self._add_new_years_day(tr("Новы год"))
-
+        name = tr("Новы год")
+        self._add_new_years_day(name)
         if self._year >= 2020:
-            self._add_new_years_day_two(tr("Новы год"))
+            self._add_new_years_day_two(name)
 
         # Orthodox Christmas Day.
         self._add_christmas_day(tr("Нараджэнне Хрыстова (праваслаўнае Раство)"))
 
         # Women's Day.
         self._add_womens_day(tr("Дзень жанчын"))
 
-        # Radunitsa (Day of Rejoicing).
-        self._add_rejoicing_day(tr("Радаўніца"))
+        if 1995 <= self._year <= 1998:
+            # Constitution Day.
+            self._add_holiday_mar_15(tr("Дзень Канстытуцыі"))
 
         # Labor Day.
         self._add_labor_day(tr("Свята працы"))
 
         # Victory Day.
         self._add_world_war_two_victory_day(tr("Дзень Перамогі"), is_western=False)
 
-        # Independence Day.
-        self._add_holiday_jul_3(tr("Дзень Незалежнасці Рэспублікі Беларусь (Дзень Рэспублікі)"))
+        # Radunitsa (Day of Rejoicing).
+        self._add_rejoicing_day(tr("Радаўніца"))
+
+        # Independence Day of the Republic of Belarus (Day of the Republic).
+        name = tr("Дзень Незалежнасці Рэспублікі Беларусь (Дзень Рэспублікі)")
+        if self._year >= 1997:
+            self._add_holiday_jul_3(name)
+        else:
+            self._add_holiday_jul_27(name)
 
-        # October Revolution Day.
-        self._add_holiday_nov_7(tr("Дзень Кастрычніцкай рэвалюцыі"))
+        if self._year >= 1995:
+            # October Revolution Day.
+            self._add_holiday_nov_7(tr("Дзень Кастрычніцкай рэвалюцыі"))
 
         # Catholic Christmas Day.
         self._add_christmas_day(tr("Нараджэнне Хрыстова (каталіцкае Раство)"), GREGORIAN_CALENDAR)
 
+        if 1992 <= self._year <= 1997:
+            # Catholic Easter.
+            name = tr("Каталiцкi Вялiкдзень")
+            self._add_easter_sunday(name, GREGORIAN_CALENDAR)
+            self._add_easter_monday(name, GREGORIAN_CALENDAR)
+
+            # Orthodox Easter.
+            name = tr("Праваслаўны Вялiкдзень")
+            self._add_easter_sunday(name)
+            self._add_easter_monday(name)
```

Comment:
@PPsyrius  @KJhellico could you double check this part?
It seems the 1992-1997 range is [applicable](https://ru.wikipedia.org/wiki/%D0%9F%D1%80%D0%B0%D0%B7%D0%B4%D0%BD%D0%B8%D0%BA%D0%B8_%D0%91%D0%B5%D0%BB%D0%BE%D1%80%D1%83%D1%81%D1%81%D0%B8%D0%B8#%D0%A0%D0%B5%D0%BB%D0%B8%D0%B3%D0%B8%D0%BE%D0%B7%D0%BD%D1%8B%D0%B5) to Monday after Easter Sunday only.

Should it also have a different name (or observed suffix), WDYT?

Reply:
@KJhellico @arkid15r I'll move Belarus's Easter to the main public list as default then 👍